build.sbt

```diff
@@ -128,6 +128,7 @@ lazy val commonSettings = clearSourceAndResourceDirectories ++ publishSettings +
   // we always assume that Java classes are standalone and do not have any dependency
   // on Scala classes
   compileOrder := CompileOrder.JavaThenScala,
+  incOptions := incOptions.value.withNameHashing(false).withAntStyle(true), // until we have class-based name hashing
```

Comment:
It think this is a poor default for new contributors, instead it should be something we enable via `local.sbt`. IIRC, I couldn't figure out an easy way to modify that setting across all projects via `local.sbt`, but with a little persistence and help it should be doable.